compair/api/course.py

```diff
@@ -78,7 +78,9 @@ def post(self):
                 new_course.end_date,
                 '%Y-%m-%dT%H:%M:%S.%fZ')
 
-        if new_course.start_date and new_course.end_date and new_course.start_date > new_course.end_date:
+        if new_course.start_date is None:
```

Comment:
you should be able to add `required=True` to new_course_parser and duplicate_course_parser `start_date` instead